### DISCUSSION TITLE
Difference between Integer and Number

### DISCUSSION THREAD

**Discussion by kaaax0815** [discordjs/discord.js#6526] [Q&A] [answered]
In Slash Command Options you have to specify the Type
Available Types: https://discord.js.org/#/docs/main/stable/typedef/ApplicationCommandOptionType

But what is the difference between `NUMBER` and `INTEGER`

The difference is `NUMBER` allow decimal places like `5.5`

`INTEGER` only allow whole numbers like `6`

**Comment by kaaax0815** [comment, marked as answer]
Answer in Post

**Comment by advaith1** [comment]
The types are explained in Discord's docs: https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-type
